src/mpl/src/thread/mpl_thread_posix.c

```diff
@@ -53,9 +57,46 @@ void MPL_thread_create(MPL_thread_func_t func, void *data, MPL_thread_id_t * idp
         pthread_attr_init(&attr);
         pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);
 
+#if MPL_THREAD_PACKAGE_NAME == MPL_THREAD_PACKAGE_UTI
+		uti_attr_t* uti_attr = NULL;
+		uti_value_t val;
+		err = uti_attr_create(uti_attr);
+		if(err) {
```

Comment:
minor nitpick - our code cleanup script will always add a space between if and conditional expression. can you fixup this and the ones below?